lib/bibliothecary/parsers/maven.rb

```diff
@@ -109,6 +115,20 @@ def self.parse_gradle_resolved(file_contents)
         end.compact.uniq {|item| [item[:name], item[:requirement], item[:type]]}
       end
 
+      def self.parse_maven_resolved(file_contents)
+        file_contents.split("\n").map do |line|
+          type_match = MAVEN_DEP_REGEXT.match(line)
+
+          next unless type_match
+          # org.springframework.boot:spring-boot-starter-web:jar:2.0.3.RELEASE:compile[36m -- module spring.boot.starter.web[0;1m [auto][m
+          {
+            name: type_match[2, 2].join(":"),
+            requirement: type_match[5],
+            type: type_match[6]
+          }
+        end.compact.uniq {|item| [item[:name], item[:requirement], item[:type]]}
```

Comment:
I really don't like the look of chaining with `do`/`end`, so I generally make another method, for something like:
```ruby
file_contents
  .lines
  .map(&method(:parse_line))
  .compact
  .uniq { ... }
```